### PR TITLE
PAN-OS DUG add/remove

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

fixes: https://github.com/demisto/etc/issues/19049

### CODE REVIEW

File: Integrations/Panorama/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## [Unreleased]
-
+  - Added 2 new commands:
+    - ***panorama-register-user-tag***
+    -***panorama-unregister-user-tag***
```
Comment (anara123) [inline Integrations/Panorama/CHANGELOG.md]:
```suggestion
    - ***panorama-unregister-user-tag***
```

**Comment by bakatzir** [comment]
@kirbles19 Please review the chagnelog, yml file
and the playbook that was added.

**Comment by kirbles19** [comment]
@yaron-libman Please take the PB for this release.

**Comment by kirbles19** [comment]
@bakatzir changelog and YML are done. @yaron-libman is taking care of the PB. I'm unassigning myself.

**Comment by bakatzir** [comment]
@yaron-libman Please see that there is 1 new PB, and 1 changed PB.
https://github.com/demisto/content/blob/f364cacd12021f9f38b72b8d80e20418e60ae405/Playbooks/playbook-NetSec_-_Palo_Alto_Networks_DUG_-_Tag_User.yml
https://github.com/demisto/content/blob/f364cacd12021f9f38b72b8d80e20418e60ae405/Playbooks/playbook-Block_Account_-_Generic.yml

**Comment by anara123** [comment]
@bakatzir please see the build, it fails on schema validation

**Comment by bakatzir** [comment]
@anara123 Schema validation will pass after the next sdk release. https://github.com/demisto/demisto-sdk/pull/190

I'm going over the PBs with yaron now.

**Comment by yaron-libman** [comment]
Sat with @bakatzir and updated texts.

**Comment by bakatzir** [comment]
@anara123 Please review again and do not force merge. I'll get it working with the sdk.

**Comment by bakatzir** [comment]
@anara123 Please review again.

**Pull request merged**